test/e2e/xsite/xsite_test.go

```diff
@@ -467,6 +468,87 @@ func TestSuspectAndHearbeatConfig(t *testing.T) {
 	expectHeartBeatConfiguration(t, testKubes["xsite2"], true, hbInterval, hbTimeout)
 }
 
+func TestMultipleSites(t *testing.T) {
```

Comment:
As we're not actually waiting for the xsite view to form, maybe we should link the issue as a comment and update the name to `TestMultipleSiteDefinitions`